sharding-orchestration/sharding-orchestration-core/src/test/java/org/apache/shardingsphere/orchestration/internal/registry/config/service/ConfigurationServiceTest.java

```diff
@@ -374,7 +373,7 @@ public void assertLoadProperties() {
     
     @Test
     public void assertGetAllShardingSchemaNames() {
-        when(regCenter.getChildrenKeys("/test/config/schema")).thenReturn(Arrays.asList("sharding_db", "masterslave_db"));
+        when(regCenter.get("/test/config/schema")).thenReturn("sharding_db,masterslave_db");
```

Comment:
Instead of using mock, is there any possible solution to start a nacos server and put the value into it?

Reply:
I read nacos's issue list, It doesn't support.

https://github.com/alibaba/nacos/issues/1926